inventory/api/os/v1/os.proto

```diff
@@ -126,6 +126,54 @@ message OperatingSystemResource {
     (buf.validate.field).ignore = IGNORE_IF_UNPOPULATED
   ]; // user-provided, human-readable description of OS
 
+  string existing_cves = 17 [
+    (ent.field) = {
+      optional: true
+      immutable: false
+    },
+    (buf.validate.field).string = {
+      max_bytes: 20000
+      pattern: "^$|^[a-zA-Z-_0-9./:;=@?!#,<>*+~()\" \\n{}\\[\\]]+$"
+    },
+    (buf.validate.field).ignore = IGNORE_IF_UNPOPULATED
+  ]; // JSON string, containig existing CVEs
+
+  string existing_cves_url = 18 [
+    (ent.field) = {
+      optional: true
+      immutable: false
```

Comment:
let's make it immutable like many other configurable fields for the OS resource.
```suggestion
      immutable: true
```